4439C4FFFEC68189.json

```diff
@@ -0,0 +1,18 @@
+++{
+ + +  "gateway_conf": {
+ + +    "gateway_ID": "4439C4FFFEC68189",
```

Comment:
Please check the content of your file, it seems to have a lot of plus signs extra. Did you copy&paste from another github change?